eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
+		"react/button-has-type": "error",
+		"react/default-props-match-prop-types": "error",
+		"react/jsx-boolean-value": ["error", "never"],
+		"react/jsx-child-element-spacing": "error",
+		"react/jsx-closing-bracket-location": "error",
+		"react/jsx-closing-tag-location": "error",
+		"react/jsx-curly-brace-presence": "error",
+		"react/jsx-curly-spacing": ["error",  {
+			"when": "always",
+			"children": true
+		}],
+		"react/jsx-equals-spacing": ["error", "never"],
+		"react/jsx-filename-extension": "error",
+		"react/jsx-first-prop-new-line": "error",
+		"react/jsx-fragments": ["error", "element"],
+		"react/jsx-handler-names": "error",
+		"react/jsx-no-bind": "error",
```

Comment:
i don't like this one. IMO is a good practice to have functions in the prop for decoupling the method and the event handling.

for instance

```
<Foo onClick={e => { e.preventDefault(); this.myBusinessLogicNoEventAware }}></Foo>
```

Reply:
I do not agree
http://shzhangji.com/blog/2018/09/13/is-it-necessary-to-apply-eslint-jsx-no-bind-rule/

Reply:
From the article above:

Conclusion
To fix jsx-no-bind we need to trade off readability, and yet the performance may not improve as much as we thought. So instead of guessing what may cause performance problem, why not program in a natural way at first, and when there occurs some noticeable performance issues, take measures, and fix them with appropriate techniques.

Reply:
I don't agree with the conclusion as well xD but his point is about readability. For me it's better to have the rule to avoid performance issues and then disable it when really necessary.

Reply:
About the decoupling, I would agree if the handler comes from props, but you can always do:
```
onClickHandler = () => {
  e.preventDefault();
  this.props.myBusinessLogicNoEventAware()
}
```

 and then 
`<Foo onClick={this.onClickHandler} />`

Reply:
renderProps with this approach is a mess too with this approach.

I would ban binds and function but leave arrow functions

Reply:
ok, I would go for `ignoreDOMComponents` and `allowArrowFunction` then